return {
    service = {
        ws = "return game:GetService(\"Workspace\")",
        plrs = "return game:GetService(\"Players\")",
        rs = "return game:GetService(\"RunService\")",
        tws = "return game:GetService(\"TweenService\")",
        tps = "return game:GetService(\"TeleportService\")",
        mkps = "return game:GetService(\"MarketplaceService\")",
        core = "return game:GetService(\"CoreGui\")",
        uis = "return game:GetService(\"UserInputService\")",
        vim = "return game:GetService(\"VirtualInputManager\")",
        rpls = "return game:GetService(\"ReplicatedStorage\")"
    },
    customfunctions = {
        reverse_boolean = "return function(t) return not t end"
    }
}
